timemachine/cpp/src/nonbonded_interaction_group.cu

```diff
@@ -351,6 +341,82 @@ void NonbondedInteractionGroup<RealType>::execute_device(
     }
 }
 
+template <typename RealType>
+void NonbondedInteractionGroup<RealType>::verify_row_indices(const std::vector<int> &row_indices) {
+    if (row_indices.size() == 0) {
+        throw std::runtime_error("idxs can't be empty");
+    }
```

Comment:
Question: Permissible for `row_indices = arange(N_)`, i.e. `col_indices.size() == 0`?

Reply:
Added in interaction group as part of [bcadefe](https://github.com/proteneer/timemachine/pull/970/commits/bcadefe01cd7941eee5d6d9cbb9c98eec053aa3e)